perfdash/metrics-downloader.go

```diff
@@ -34,39 +34,32 @@ import (
 	"k8s.io/kubernetes/test/e2e/perftype"
 )
 
-// GoogleGCSDownloaderOptions is an options for GoogleGCSDownloader.
-type GoogleGCSDownloaderOptions struct {
+// DownloaderOptions is an options for Downloader.
+type DownloaderOptions struct {
 	ConfigPaths        []string
 	GithubConfigDirs   []string
 	DefaultBuildsCount int
-	LogsBucket         string
-	LogsPath           string
-	CredentialPath     string
 	// Development-only flag.
 	// Overrides build count from "perfDashBuildsCount" label with DefaultBuildsCount.
 	OverrideBuildCount bool
 }
 
-// GoogleGCSDownloader that gets data about Google results from the GCS repository.
-type GoogleGCSDownloader struct {
-	GoogleGCSBucketUtils *bucketUtil
-	Options              *GoogleGCSDownloaderOptions
+// Downloader that gets data about results from a storage service (GCS) repository.
+type Downloader struct {
+	MetricsBkt     MetricsBucket
```

Comment:
I'd probably spell this out for consistency. It's definitely a go practice to shorten variables, but this is typically done for local variables, rather than types, interfaces, and external functions.